api/proto/config/appconfig.proto

```diff
@@ -26,6 +26,47 @@ enum MetaDataType {
   MetaDataDriveMultipart = 3; // Process multipart MIME for application
 }
 
+// The information about a snapshot
+message AvailableSnapshot {
+  string id = 1;
+}
+
+// The snapshot information for an Application Instance
+message SnapshotConfig {
+  // activeSnapshot is the id of the snapshot which is currently used.
```

Comment:
Makes sense to follow buflint: buflint:Field name "activeSnapshot" should be lower_snake_case, such as "active_snapshot".

Reply:
We have a mix of lower_snake_case and camelCase — all the fields before 17.
If we want to follow the recommendation, do we have to fix the previous fields?